PluginJira/DataContracts/ApplicationEndpointWrapper.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+
+namespace PluginJira.DataContracts
+{
+    public class ApplicationEndpointWrapper
+    {
+        public string key { get; set; }
```

Comment:
for all items in this class prefer to use json labels over lowercase class names and keep all class variables in PascalCase
```
using Newtonsoft.Json;

...

[JsonProperty("key")]
public string Key { get; set; }
```